.github/workflows/cd-static-cdn.yml

```diff
@@ -8,15 +8,29 @@ on:
 jobs:
   deploy_linux:
     runs-on: ubuntu-latest
-  
+
     steps:
     - name: Checkout Branch
       uses: actions/checkout@master
 
-    - name: Upload static assets to CDN 
+    - name: Upload static assets to CDN
       uses: lauchacarro/Azure-Storage-Action@master
       with:
         folder: 'sites/site-utilities/statics/assets'
         connection-string: ${{ secrets.AZURE_CONNECTION_STRING_STORAGE }}
         public-access-policy: Blob
-        blob-container-name: 'assets'
\ No newline at end of file
+        blob-container-name: 'assets'
+
+  notify:
+    runs-on: ubuntu-latest
+    needs: deploy_linux
+
+    steps:
+    - name: Notify on Discord
+      uses: appleboy/discord-action@master
+      with:
+        webhook_id: ${{ secrets.DISCORD_NOTIFICATION_WEBHOOK_ID }}
+        webhook_token: ${{ secrets.DISCORD_NOTIFICATION_WEBHOOK_TOKEN }}
+        color: '#DE2D6D'
+        username: 'FAST DevOps CDN Bot'
+        message: 'Deployment has completed to CDN on https://github.com/microsoft/fast/actions/workflows/cd-static-cdn.yml'
```

Comment:
Do these (various releases) all run simultaneously? Do we have any concerns about the bots being noisy?

Reply:
This notifies our private #ms-internal-notifcations channel which may be noisy though only delivery workflows trigger notifications.

Pros:
* I will be notified on delivery and can easily click a link to view pass/fail of deployments to proactively resolve to breaks or blocking issues.
* Anyone can easily click and become familiar with how we release and in what workflows
* Elevates visibility for leadership curious on what's most recently deployed.
* Can easily be ignored in this channel for those not concerned.
* It's almost like a breaking news feed, I suppose.

Reply:
I believe we could also do some updates to include the event, the branch, and path whereby the path would could be restricted to only deploy if files required or dependencies of a site are changed.  So this would reduce noise if it became a problem.

Reply:
Thanks for the follow up @awentzel. This is probably fine for now, but I do like the enhancement above. Would be great over time to have builds conditional on changed dependencies for sites and perhaps packages, etc.